SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Scope of security vulnerabilities
+
+uv is a Python package manager. Due to the design of the Python and its packaging ecosystem, there
```

Comment:
"the Python" seems off. "Due to the design of Python packaging ecosystem"?

Reply:
Ah, edits.